react-commenting.js

```diff
@@ -90,16 +97,19 @@ class Board extends Component{
 
           <div className="statusDiv">
             {/* 9) Here: */}
+            //Current player is part of this destructuring state.
             The Current Player is: { currentPlayer }
           </div>
 
           <div className="statusDiv">
             {/* 10) Here: */}
+            //The winner is part of this destructuring state. 
             The Winner is: { winner }
```

Comment:
Same thing here. We did destructure winner out of state, but this is displaying the winner to the user.